frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.tsx

```diff
@@ -90,6 +94,21 @@ const CreateVMWizardComponent: React.FC<CreateVMWizardComponentProps> = (props)
   const [dataVolumePVCs, dataVolumePVCsLoaded, dataVolumePVCsLoadError] = usePVCBaseImages(
     commonTemplates,
   );
+  const activeNamespace = getActiveNamespace();
+  const nadsResource = React.useMemo(
+    () => ({
```

Comment:
this is not a place for listening to resources as it is inefficient and adds more complexity. Please add a nads resource to CreateVMWizardPageComponent.